src/main/java/org/ballerinalang/docgen/docs/html/HtmlDocumentWriter.java

```diff
@@ -274,6 +275,19 @@ private void writeHtmlDocument(Object object, String templateName, String absolu
             }
         }
     }
+    
+    /**
+     * Check whether an annotation has the provided fully qualified name.
+     * 
+     * @param annotationName fully qualified name
+     * @param annotation Annotation to compare with
+     * @return Flag indicating whether the annotation has the provided fully qualified name
+     */
+    private boolean isNameEqual(String annotationName, AnnotationAttachment annotation) {
+        String [] pkgAndName = annotationName.split(":");
```

Comment:
@SupunS It is discussed to create an issue and merge this. Please create an issue.